addons/mail/static/src/core/web/activity_list_popover_item.js

```diff
@@ -76,11 +76,22 @@ export class ActivityListPopoverItem extends Component {
         return this.props.activity.state !== "done" && !this.hasFileUploader;
     }
 
+    get hasCancelButton() {
+        return this.props.activity.state !== "done";
+    }
+
     onClickEditActivityButton() {
         this.props.onClickEditActivityButton();
         this.props.activity.edit().then(() => this.props.onActivityChanged?.());
     }
 
+    unlink() {
```

Comment:
Ordering alphabetically :slightly_smiling_face: Should be moved under `onFileUploaded`